app/src/main/java/com/example/android/architecture/blueprints/todoapp/TodoNavigation.kt

```diff
@@ -102,5 +105,11 @@ class TodoNavigationActions(private val navController: NavHostController) {
                 if (taskId != null) "$it?$TASK_ID_ARG=$taskId" else it
             }
         )
+
     }
+
+    fun onQuit() {
+        System.exit(0)
```

Comment:
Never do this in an android app, it can mess up with activities lifecycle.


Also in this instance, there is really no need.

Reply:
ok so on my phone (Pixel) I just figured out the gesture for Back is swipe up from bottom. 

I wasn't that familiar with the gestures. 

I am running the new android beta